### ISSUE TITLE
Best advice for wind and trees?

### ISSUE THREAD

**Issue by talormanda** [ccrisan/motioneyeos#1723]
During the winter, I get a lot of shadow movement from trees losing their leaves and the wind. I tried to use the Smart Mask Type but it kept triggering the same areas over and over. Is there anything I am missing or something I can do to better combat this issue other than disabling entire sections of the camera?

**Comment by mb39924** [comment]
Have you tried if turning the Despeckle Filter (under motion detection) on has any effect?

**Comment by talormanda** [comment]
@mb39924 - do you know what that does? (tldr?)

**Comment by mb39924** [comment]
Yes I know but a had the same problem and it helped reduce the notifications.

**Comment by talormanda** [comment]
@mb39824 - I'm asking what it does to explain to me, not if you know what the concept is.

**Comment by jasaw** [comment]
@talormanda Despeckle filter allows you to erode and dilate the pixels according to your needs. Read http://emit.demon.co.uk/motion/ if you want to know exactly what it does.
I suggest you modify the despeckle_filter field in thread-1.conf file by hand if you want anything other than `EedDl` because MotionEyeOS Despeckle filter only sets `EedDl`.

From my personal experience, I find `EeEeEedDdDdDl` works best for my set up. I get minimal false triggers from wind and trees. Moving clouds on the other hand is still giving me grieve.

**Comment by AndyMAu** [comment]
I use masking(editable mode) on some trees in front of my cameras. It is helpful for me as it reduces the overall area which detects movement.

**Comment by RPiSpy** [comment]
I use the "custom mask" and just mask out the areas with trees and bushes. Obviously you can only mask out so much before you miss a person but I've managed to get a sensible balance. If you mask out a lot you can up the sensitivity in the remaining areas to ensure any movement in the "quiet" areas get caught.
